wolfssl/internal.h

```diff
@@ -1654,6 +1654,15 @@ enum Misc {
     READ_PROTO         = 0     /* reading a protocol message */
 };
 
+#ifndef WOLFSSL_NO_TLS12
+#ifdef WOLFSSL_SHA384
+    #define HSHASH_SZ WC_SHA384_DIGEST_SIZE
```

Comment:
We likely already have a macro or enum for this. If not please document what this value is and put with other TLS related settings in this file.

Reply:
Its moved from `src/tls.c` to here, but I now noticed it is no longer required. Will move it back.

Reply:
Done.